core/org.eclipse.cdt.core/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.cdt.core; singleton:=true
-Bundle-Version: 8.4.0.qualifier
+Bundle-Version: 8.3.0.qualifier
```

Comment:
FYI @jjohnstn I put the bundle version back to 8.3 as it was already bumped in this release in https://github.com/eclipse-cdt/cdt/blame/1e04efa017cabdf35ce029b5b71cc78ee008ca41/core/org.eclipse.cdt.core/META-INF/MANIFEST.MF